tools/bug-report/pkg/bugreport/bugreport.go

```diff
@@ -303,3 +325,25 @@ func BuildClientsFromConfig(kubeConfig []byte) (kube.Client, error) {
 	}
 	return clients, nil
 }
+
+func logAndPrintf(format string, a ...interface{}) {
```

Comment:
I think you can use `clog` here.
https://github.com/istio/istio/blob/c574f8ce63b2bd0fa0531320ba0e9265368c07bc/operator/pkg/util/clog/clog.go#L30

Reply:
Why do we need log and printf? I thought operator only needed it since it has two modes (in cluster server mode and cli). This only has a CLI mode? Seems like we should log or print, not both

Reply:
Oh we are writing to a file. This still seems a bit off - the goal is to get the output into a command line and into the file right? Why don't we just configure the log to write to a io.Writer which writes to stdout and to the file? Then we don't need to pass around this logging object and can just use plain old `log.Info`. This PR is only using logAndPrintf in like 5 places so surely we are missing quite a few places, which would be impossible with this approach?

Reply:
the logging output is too verbose to go to console. conversely, the console output does not contain all the info we need to debug problems with the tool itself (or interpret the archive in some cases).
the basic problem is we somehow need to mark some specific log statements as going to both console and file and this seems like a fairly simple and comprehensible way of doing that.

Reply:
That's fair, I didn't realize we had that distinction